src/components/payment/firstSignUp.module.css

```diff
@@ -1,4 +1,8 @@
 .container{
+    margin: 0;
+  position: absolute;
```

Comment:
use flex not this way. it will give problems

Reply:
display : flex
flex-direct:row (or column) check
align-items:center
justify-content: center